sys/linux/dev_usb_hiddev.txt

```diff
@@ -0,0 +1,90 @@
+# Copyright 2019 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+include <uapi/asm/ioctl.h>
+include <uapi/linux/hiddev.h>
+
+resource fd_hiddev[fd]
+
+syz_open_dev$hiddev(dev ptr[in, string["/dev/usb/hiddev#"]], id intptr, flags flags[open_flags]) fd_hiddev
+
+write$hiddev(fd fd_hiddev, data buffer[in], len bytesize[data])
```

Comment:
Is this data opaque for kernel code, or there is some format expected? If there is some format, add a TODO.

Reply:
`write` is actually not implemented, I'll remove it.